azext_iot/iothub/_help.py

```diff
@@ -105,7 +105,7 @@ def load_iothub_help():
         examples:
         - name: Invoke root level command "reboot" which takes a payload that includes the "delay" property.
           text: >
-            az iot hub digital-twin invoke-command --command-name reboot -n {iothub_name} -d {device_id} --payload '{"delay":5}'
+            az iot hub digital-twin invoke-command --command-name reboot -n {iothub_name} -d {device_id} --payload "5"
```

Comment:
We should make sure this works with a PnP device client sample

Reply:
thanks will test it out!

Reply:
@digimaun for PnP device client sample do you mean device defined using something like this https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/TemperatureController.json? Just to confirm if I'm verify using the right way. I firstly used `az iot device simulate -n dfsdf -g jiacjutest -d device1 --model-id 'dtmi:com:example:TemperatureController;1'` to simulate the device using corresponding model id, and then send the command using `az iot hub digital-twin invoke-command --command-name reboot -n dfsdf -d device1 --payload "5"`, and the response I got is:
{
  "payload": {
    "methodName": "reboot",
    "methodRequestId": "1",
    "methodRequestPayload": 5
  },
  "status": "200"
}

would this wipe your concern?😋

Reply:
We discussed this on friday and the result is to have two examples, one with the older json and another with the new one with the help text explaining what the difference is between the two (as in what models would work with them)

Reply:
@vilit1 example updated!😀

Reply:
The TemperatureController.json dtdl. delay is an integer. we should be sending 5 directly instead of a string.

Reply:
resolved offline